scripts/plugin-godocs.go

```diff
@@ -53,6 +57,31 @@ func docHTML(text string) string {
 	return buf.String()
 }
 
+func removeDuplicates(array []string) []string {
+	keys := make(map[string]struct{})
+	set := []string{}
+	for _, element := range array {
+		if _, value := keys[element]; !value {
```

Comment:
I'd suggest using the variable name `ok` instead of `value` to be more idiomatic.

Reply:
Ok